### PR TITLE
Added : Button and rating

### PR DESCRIPTION
**Pull request by aiman-ansari**
Screenshot (26)
Screenshot (27)
Screenshot (28)
Screenshot (29)
Screenshot (30)
Screenshot (31)
Screenshot (32)
Screenshot (33)
Screenshot (34)

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Good to merge

File: src/components/Button/Button.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    min-width: 8rem;
+    cursor:pointer;
+    align-items: center;
+    border:none;
+    padding:0;
+    color:var(--white);
+    align-items: center;
+    font-size:0.9rem;
+    border-radius:25px;
+    font-weight: 600;
```
Comment (ShrutiLakhani) [inline src/components/Button/Button.css:39]: Avoid using px here

**Review by ShrutiLakhani — APPROVED** [review]
LGTM!

File: src/components/Button/Button.html [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+&lt;button class=&quot;btn-floating-icon btn-light box-shadow&quot;&gt;
+    &lt;i class=&quot;fa fa-plus icon-light&quot;&gt;&lt;/i&gt;
+&lt;/button&gt;
+
+                    </code>
+                </pre>
+                <div id="gradient-btn">
+                    <h2>Gradient Button</h2>
+                    <p class="mt-1">You can use gradient button by simply adding <span class="purple-text">btn-gradient-pink</span> for pink and orange combination</p>
+                    <p>and <spna class="purple-text">btn-gradient-blue</spna> for blue and teal color combination</p>
```
Comment (SinchanaMS) [inline src/components/Button/Button.html:198]: span tag is spelt wrong here

File: src/components/Rating/Rating.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.rating ul li{
+    display: inline-block;
+    padding: 1px;
+
+    /* ba */
+}
+.bi-star-fill,
+.bi-star-half,
+.bi-star{
+    color:rgb(255, 181, 43)
```
Comment (SinchanaMS) [inline src/components/Rating/Rating.css:10]: use colour variable here..

File: src/components/Button/Button.html [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+    <main>
+        <div class="container-section">
+            <h2>Button</h2>
+            <p class="mt-1 mb-1 bold-text">Use these custom button styles for action in forms, dialogs and more.</p> 
+        <hr>
+            <section>
+            <div id="simple-button">
+                <h2 class="mt-1">Simple Button</h2>
+                <p class="mt-1">Customise your button by adding<span class="purple-text">.btn</span>class to button.</span>
+                    <br>There are 4 deffierent types of colors.<br>
```
Comment (SinchanaMS) [inline src/components/Button/Button.html:60]: different is spelt wrong.

**Pull request merged**